core/runtime/src/main/java/io/quarkus/runtime/ApplicationLifecycleManager.java

```diff
@@ -183,7 +183,7 @@ public static void run(Application application, Class<? extends QuarkusApplicati
                     } else {
                         for (Integer port : ports) {
                             applicationLogger
-                                    .warnf("Use 'netstat -anop | grep %d' to identify the process occupying the port.", port);
+                                    .warnf("Use 'ss -anop | grep %d' to identify the process occupying the port.", port);
```

Comment:
```suggestion
                                    .warnf("Use 'ss -anop | grep %d' or 'netstat -anop | grep %d' to identify the process occupying the port.", port);
```

Reply:
Done.

Thanks @gsmet for the elegant suggestion. 

BSD has another implementation for the `netstat` command. I thought in creating a new conditional block for linux (`else if (IS_LINUX)`) but I prefer your solution.